README.md

```diff
@@ -104,13 +104,23 @@ The configuration for queues/topics can be changed or reset to default on **Demo
 | Inventory service request queue   | `inventory.request`                                      |
 | Inventory service response queue  | `inventory.response`                                     |
 
+Part of the configuration can be changed in **application.properties** file.
+
 **Configuration details for external Kafka server (default)**
 
 |    Option   |             Value              |
 |-------------|--------------------------------|
 | Broker URL  |       `localhost:9092`         |
 | Group ID    |       `inventory-operation`    |
 
+This configuration can be changed in **application.properties** file.
+
+> All messages from this application are sent to the partition 0 in the topic,
+> it means the consumer should listen on the partition 0 of the topic.
+
+> Since the messages will only be consumed once by consumers with the same group ID,
+> pay attention to use default Group ID in external application(like SOAtest & Virtualize).
+
```

Comment:
For simplicity, messages produced or consumed in this application will be in partition 0 of the topic.
When the expected behavior is to consume the message from both inside this application and outside this application (e.g. in SOAtest & Virtualize), make sure to use different group ID for the external consumer.

Reply:
I think there is no case we want to use the same group ID in SOAVirt.